__tests__/app.test.js

```diff
@@ -92,6 +92,45 @@ describe("/api/reviews/:review_id", () => {
       });
   });
 });
+describe.only("/api/reviews/:review_id/comments", () => {
+  test("GET - 200: Returns the review with the correct ID", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBe(3);
+        expect(body).toBeSortedBy("created_at", { descending: true });
+        body.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              review_id: expect.any(Number),
+            })
+          );
+        });
+      });

```

Comment:
We need one more 'happy path' test here. It is possible for a review to have no associated comments. In this case i would expect an empty array returned with a 200 status